.changeset/lemon-dragons-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@module-federation/enhanced': patch
+---
+
+refactor container plugin single runtime logic and fix types
```

Comment:
because the file has been formatted , and i only watch the change is that `@ts-ignore` has been removed , has anything change i miss ?

Reply:
ahh yes you are correct

Reply:
i removed it